blockchain/check-backup.js

```diff
@@ -67,23 +66,35 @@ stream.on("finish", async () => {
 
     const hash = await createHash(extractPath);
     const isValidMetadataFile = config.DirectoryHash === hash;
+
+    //Check for major version compatibility
+    const compatibleVersions =
+      config.hasOwnProperty("Version") &&
+      config.Version.split(".")[0] === version.split(".")[0];
+
     if (isValidMetadataFile) {
-      console.log(`The provided backup file is valid\n`);
-      console.log(`No updated backup is created`);
+      console.log("The provided backup file is valid\n");
+      console.log("No updated backup is created");
       process.exit(0);
+    } else if (compatibleVersions) {
+      console.log("The provided backup is from a prior major version.\n");
```

Comment:
So the Versions are compatible according to line 79 but they are not in line 80?